crates/nu-command/src/viewers/table.rs

```diff
@@ -878,9 +937,22 @@ fn convert_to_table(
         data.push(row);
     }
 
-    let count_rows = data.len();
+    // All the computations are parallelised here.
+    // NOTE: It's currently not possible to Ctrl-C out of this...
+    let mut cells: Vec<Vec<_>> = Vec::new();
+    cells.par_extend(data.into_par_iter().map(|row| {
+        let mut new_row = Vec::new();
```

Comment:
Hi there;

I wonder if you could use `Vec::with_capacity` and use index instead of `extend`.
As I see `collect_into_vec` might do exactly that?

Reply:
When I was writing those lines I was making some misassumptions about what Rayon methods would guarantee preserved ordering… I have a better sense of it now and can write this a bit different.